packages/dual-channel/docs/zh-tw/mannual.md

```diff
@@ -0,0 +1,193 @@
+2. 在第一個欄位裡（見圖 1-4），填入你的 Spreadsheet 的網址或是 Spreadsheet ID（見圖 1-3）
+3. 依序填入你想要出現在網頁上的工作表 ID（Sheet ID）（見圖 1-5）（註 1）
+4. 若不填工作表 ID，程式會自動抓取 Spreadsheet 上**所有的章節**。
+5. 點選 `BUILD` 按鈕
+6. 稍等一下，頁面會產生出「鑲嵌程式碼」
+7. 點選`一鍵複製`按鈕，即可將程式碼貼到你的網頁裡。
+
+![圖1-3](../assets/figure-1.3.png)
+
+_圖 1-3_
+
+![圖1-4](../assets/figure-1.4.png)
+
+_圖 1-4_
+
+＊註 1：點了 Spreadsheet 下方的工作表後（見圖 1-5），該 Spreadsheet 網址的最後會出現 `#gid=492967532`，`492967532`是該工作表的 ID，將此 ID 填入產生器中，即可產生該章節的內容。
+
+![圖1-5](../assets/figure-1.5.png)
+
+_圖 1-5_
+
+### 製圖注意事項
+
+#### 桌機版的圖
+
+請以寬 415px，長 453px 的長方形來設計。
+
+#### 手機版的圖
+
+請以正方形來設計，例如：長 375px、寬 375px。
+
+### 常見問題
+
+#### 1. 我可以將圖片上傳到哪裡呢？
+
+可以將圖片上傳到有提供圖片託管服務的網路空間，並取得圖片的公開網址。
+
+下面是幾個推薦用來上傳圖片的平台以及操作指引：
+
+- [Google 雲端硬碟](https://github.com/twreporter/orangutan-monorepo/tree/master/packages/scrollable-image/docs/zh-tw/google-drive-hosting.md)
+- [Dropbox](https://github.com/twreporter/orangutan-monorepo/tree/master/packages/scrollable-image/docs/zh-tw/dropbox-hosting.md)
+- imgur.com
+
+#### 2. 若是我不熟 HTML 和 CSS，我該如何使用「左右互搏」？
+
+現階段可以參考[Dual Channel 測試資料 - Google 試算表](https://docs.google.com/spreadsheets/d/1Ppisv4HTZHYMp95umgCoADNuP1PSkL-t9na-5lRIqSY/edit#gid=2143095237)來進行修改。
+我們會根據使用者的回饋進行調整，希望之後的版本可以滿足你的需求，
+為了進一步了解你的需求，請你給予我們回饋（請見下方[回饋](#回饋)）。
+
+#### 3. 若是圖片的大小或是長寬比不同於放圖的區塊，導致圖的呈現有些扭曲或是圖沒有被完整呈現，我該怎麼辦？
+
+由於我們在 Spreadsheet 中的**圖片**欄中，放入的值是 html code，因此可以透過 css 來調整。
+舉例而言，若我的**圖片**欄中的值為
+
+```html
+<picture
+  ><source type=“image/svg+xml” media=“(orientation: portrait)”
+  srcset=“https://www.twreporter.org/images/responsive/20200306072512-9fa12d4cc8.svg”
+  /><source type=“image/svg+xml” media=“(orientation: landscape)”
+  srcset=“https://www.twreporter.org/images/responsive/20200306072512-f3dcc75cb7.svg”
+  /><img alt=“”
+  src=“https://www.twreporter.org/images/responsive/20200306072512-9fa12d4cc8.svg”
+  style=“width:100%;height:100%” /></picture
+>
+```
+
+那我可以在 <img> 標籤中，加入 `object-fit:contain` 的方式，讓圖等比例縮放至區塊裡；亦可以使用 `object-fit:cover` 的方式，讓圖撐滿區塊，但圖的周圍會被自動裁掉。
+以上述範例來改，最後的結果會是，
+
+```html
+<picture
+  ><source type=“image/svg+xml” media=“(orientation: portrait)”
+  srcset=“https://www.twreporter.org/images/responsive/20200306072512-9fa12d4cc8.svg”
+  /><source type=“image/svg+xml” media=“(orientation: landscape)”
+  srcset=“https://www.twreporter.org/images/responsive/20200306072512-f3dcc75cb7.svg”
+  /><img alt=“”
+  src=“https://www.twreporter.org/images/responsive/20200306072512-9fa12d4cc8.svg”
+  style=“width:100%;height:100%;object-fit:contain;” /></picture
+>
+```
+
+#### 4. 如果我用 infogram 產生圖（或圖表），我可以套用在「左右互搏」之中嗎？
+
+Yes，你可以這麼做，你可以將 infogram 產生給你的鑲嵌程式碼，輸入在 Spreadsheet 的**圖片**欄裡，
+如此一來，即可套用 infogram 在你的作品中。
+
+#### 5.「左右互搏」 支援哪些瀏覽器？
+
+我們主要用來開發瀏覽器是 Google Chrome，經過測試，在 Firefox、Safari、Internet Explorer (IE 10 以上，含 IE 10)、Edge 都能夠正常運作。
+
+#### 6.「左右互搏」是否免費提供商業使用？
+
+是，「左右互搏」使用 MIT 授權條款，可供商業使用且不需要支付任何費用，詳細資訊請見[授權條款頁](https://github.com/twreporter/orangutan-monorepo/blob/master/LICENSE)。
```

Comment:
update this as https://github.com/twreporter/orangutan-monorepo/pull/26#discussion_r407445343 ?